### DISCUSSION TITLE
iPhone 端显示 No internet connection detected

### DISCUSSION THREAD

**Discussion by x113773** [getlantern/lantern-forum-cn#874] [General]
今天更新最新版本后，刚开始还能用一段时间，然后就一直连不上网络

**Comment by MaxMKMax** [comment]
@x113773 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by x113773** [reply]
iphone端反馈问题也是报错，用PC端提交的报告：77088483

**Reply by MaxMKMax** [reply]
@x113773 您好. 请问您可以提供我截图吗? 还是录个影片给我. 因为PC端的报告没办法提供咨询. 需要收集的log或截图.

技术团队需要这个来修复问题. 谢谢您. 麻烦您了!

**Comment by garyxuan** [comment]
我也是这个问题

**Reply by MaxMKMax** [reply]
@garyxuan 您好. 请问您可以提供我截图吗? 还是录个影片给我. 技术团队需要这个来修复问题. 谢谢您. 麻烦您了!

**Comment by cyrilpetrilli** [comment]
me too   ios端口问题反馈邮件好像也不通畅 没有收到 ticket  NO.

**Reply by MaxMKMax** [reply]
@cyrilpetrilli 您好. 请问您可以提供我截图吗? 还是录个影片给我. 技术团队需要这个来修复问题. 谢谢您. 麻烦您了!

**Comment by cyrilpetrilli** [comment]
我把过程录屏发过来 请查收

Mike K ***@***.***> 于2024年9月25日周三 01:39写道：

**Comment by cyrilpetrilli** [comment]
我个人的猜想是在电信服务提供商的机房那边有什么问题 不一定是软件的问题

Mike K ***@***.***> 于2024年9月25日周三 01:39写道：

**Comment by symPhysics** [comment]
有人解决这个问题了吗？   我的iphone也是No internet connectionn detected.  
andrioid和mac用的还正常，就ios连不上

**Reply by MaxMKMax** [reply]
@symPhysics 您好, 请问您能用iOS的手机在没有网络连接的情况下提交应用内报告吗? 
虽然这个问题很奇怪, 但我们想尽可能收集到资料帮你们解决问题. 只能用iOS手机提交哦.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by symPhysics** [reply]
ticket号码： 80380838
目前版本是 version 8.0.2 Build50 SDK7.6.107。

另外遇到一个问题：在我选择报告问题的类型选择“无法连接设备”时，点击发送报告后出现了strcov.Atoi:parsing "": invalid syntax的错误。所以我改选了other选项，这样没有报错。

**Comment by MaxMKMax** [comment]
@symPhysics 您也顺便检查一下您是否是最新版本.

**Comment by MaxMKMax** [comment]
@symPhysics 好的. 谢谢您的帮忙. 非常感谢您支持蓝灯. 我们会尽所能去优化产品的.
